site/pages/wrap-eth.js

```diff
@@ -0,0 +1,207 @@
+import { useState, useContext, useEffect } from 'react'
+import Big from 'big.js'
+import { useWeb3React } from '@web3-react/core'
+import Layout from '../components/Layout'
+import useTranslation from 'next-translate/useTranslation'
+import Input from '../components/Input'
+import Button from '../components/Button'
+import PureContext from '../components/context/Pure'
+import { fromUnit, toUnit, toFixed } from '../utils'
+import { useRegisterToken } from '../hooks/useRegisterToken'
+import { useBalance } from '../hooks/useBalance'
+
+const Operation = {
+  Wrap: 1,
+  Unwrap: 2
+}
+
+const useTemporalMessage = function () {
+  const state = useState()
+  const [text, setText] = state
+  useEffect(
+    function () {
+      if (!text) {
+        return
+      }
+      const CLEANUP_TEXT_MS = 5000
+      const timeoutId = setTimeout(() => setText(undefined), CLEANUP_TEXT_MS)
+      return () => clearTimeout(timeoutId)
+    },
+    [text, setText]
+  )
+
+  return state
+}
```

Comment:
How do we do this in the other mini-apps?

Reply:
according to the code (I can't really test it as I don't have valid inputs for the other apps) we just clear the messages after the user connects / disconnects their wallet, or if the user interacts with an input. Otherwise, the message remains there forever. That's for the 3 mini-apps. Would you prefer I do it with that approach instead of clearing the feedback after N time?

Reply:
Simpler seems better. Consistency is also important but don't change anything now. Was just asking.